composer.json

```diff
@@ -2,7 +2,7 @@
   "name": "reach/reachpayment",
   "description": "",
   "type": "magento2-module",
-  "version": "2.3.5",
+  "version": "2.3.6",
```

Comment:
do you think these changes and the MAG-85 changes will go out together? I would assume, just thinking that if they do go out together then we would only need the version number bumped in this code review

Reply:
Good point. Both MAG-39 and MAG-85 can go out together. If not then MAG-85 should be released first. That said how about considering Adam's view too as he will be testing and both being together may make his life difficult. Let me talk and then adjust the version for mag-85 and mag-39 accordingly. Does that sound good? Thanks!